modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/test/js/components/__snapshots__/index.es.js.snap

```diff
@@ -257,6 +257,13 @@ exports[`index renders 1`] = `
         </div>
       </div>
     </div>
+    <div
+      class="sidebar-container sidebar-reports"
+    >
+      <div
+        class="sidebar sidebar-light"
+      />
+    </div>
```

Comment:
@marcosapmf and @carolmariaabb, let's make a deal: for new front-end tests we won't use snapshots, however for the existent tests using it we'll keep them, ok?

Reply:
Ok @natocesarrego !

Reply:
Ok @natocesarrego  :D